pkg/cmd/cli/cli_changefeed_helper.go

```diff
@@ -41,6 +41,7 @@ func confirmLargeDataGap(cmd *cobra.Command, currentPhysical int64, startTs uint
 		var yOrN string
 		_, err := fmt.Scan(&yOrN)
 		if err != nil {
+			cmd.Printf("Received invalid input: %s, abort changefeed %s\n", err.Error(), command)
```

Comment:
The main problem in the issue is the unreadable error msg , such as the table name?